pkg/icingadb/db.go

```diff
@@ -98,7 +98,7 @@ func (db *DB) BuildUpdateStmt(update interface{}) string {
 		`UPDATE %s SET %s WHERE id = :id`,
 		utils.TableName(update),
 		strings.Join(set, ", "),
-	)
+	), len(columns)
```

Comment:
With `:id` in the `WHERE` clause, there is one more placeholder.

Reply:
True!